tests/test_sets/fixed_distributional_test_dataset.csv

```diff
@@ -0,0 +1,501 @@
+beta,beta_std,chi2,chi2_std,exponential,exponential_std,gamma,gamma_std,norm,norm_std,poisson,poisson_std,uniform,uniform_std
```

Comment:
It is being used currently for the test_dataset_util tests in test_infer_distribution_parameters and test_validate_distribution_parameters... but if you know of a way to load in dataset from a JSON file instead of csv, than these are not needed!